ngraph/core/include/ngraph/op/gather.hpp

```diff
@@ -64,5 +64,47 @@ namespace ngraph
                                      const HostTensorVector& inputs) const;
             };
         } // namespace v1
+
+        namespace v7
+        {
+            /// \brief Gather slices from axis of params according to indices
+            class NGRAPH_API Gather : public Op
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+                static const int64_t AXIS_NOT_SET_VALUE = std::numeric_limits<int64_t>::max();
```

Comment:
I propose to hide `AXIS_NOT_SET_VALUE` from the header.

Reply:
moved to private